.github/workflows/nightly.yml

```diff
@@ -48,15 +48,7 @@ jobs:
           sudo apt-get install -y \
             crossbuild-essential-arm64 \
             crossbuild-essential-s390x \
-            crossbuild-essential-ppc64el \
-            libseccomp-dev:amd64 \
-            libseccomp-dev:arm64 \
-            libseccomp-dev:s390x \
-            libseccomp-dev:ppc64el \
```

Comment:
We don't need the libseccomp-dev packages anymore either?

Reply:
We don't need libseccomp-dev since https://github.com/containerd/containerd/commit/7332e2ad2e5f064b3de9f364f38f3d4ad10db935